### ISSUE TITLE
Ability to change the default url text

### ISSUE THREAD

**Issue by danielcgold** [ayastreb/jekyll-maps#34]
When you supply a URL, the text shows up in the dialog as "View". It's be cool to have an extra param in the data like, url_text to be able to customize the popup.

So like:

```
- title: Place Name
  url: http://www.place.net
  url_text: View place name
  latitude: 6.206731
  longitude: -75.5677747
```

**Comment by ayastreb** [comment]
Implemented in v2.3.0

**Comment by danielcgold** [comment]
Yay it works! 🎉
